tests/testthat/test-step-call.R

```diff
@@ -161,3 +161,69 @@ test_that("unique is an alias for distinct", {
   dt <- lazy_dt(data.table(x = c(1, 1)))
   expect_equal(unique(dt), distinct(dt))
 })
+
+# drop_na ------------------------------------------------------------------
+
+test_that("empty call drops every row", {
+  df <- lazy_dt(tibble(x = c(1, 2, NA), y = c("a", NA, "b")), "DT")
+  exp <- tibble(x = c(1), y = c("a"))
+  step <- drop_na(df)
+  res <- as_tibble(step)
+  expect_equal(show_query(step), expr(na.omit(DT)))
+  expect_equal(res, exp)
```

Comment:
I _think_ this is a little easier to follow:

```suggestion
  tb <- tibble(x = c(1, 2, NA), y = c("a", NA, "b")
  dt <- drop_na(lazy_dt(tb, "DT"))
  expect_equal(show_query(dt), expr(na.omit(DT)))
  expect_equal(as_tibble(step), tb[1, ])
```